tests/index.spec.ts

```diff
@@ -403,7 +403,8 @@ describe('prisma-relay-cursor-connection', () => {
     ]
 
     test.each(VALID_CASES)('%s', async (name, connectionArgs) => {
-      const result = await findManyCursorConnection(
+      const result = await findManyCursorConnection<Todo, { id: string }, Todo & { extraNodeField: string },
```

Comment:
TS cannot pick up Custom Edge and Node fields if we don't specify generics here. I imagined it could infer these from pOptions argument but couldn't make it work. It would be awesome if we can make this work without specifying generics but I am not sure if this is a limitation of TS.

Reply:
Huh, I would have thought that TS could infer that, but I could not get it to work either. It would be awesome, but I think this is fine for now 👍